deploy/template.env

```diff
@@ -0,0 +1,37 @@
+# ##############################################################################
+# Wikibase Suite
+#
+# This file is a template for the .env file used by the Wikibase Suite Docker
+# Compose setup.
+#
+# WARNING: Do not add comments on the same line as env vars, as in some
+# environments they will be included in the var!
+#
+# shellcheck disable=2034 # do not warn about unused variables
+# ##############################################################################
+
+# Public Hostname Configuration.
+# Define the DNS Domain Names for your Wikibase Suite services. The DNS entries
```

Comment:
Suggested change to passive voice, and somewhat more clear:

"These domain names for your Wikibase Suites services should be configured on your DNS server to point to the public IP address of the server being deployed to."

Reply:
sold